src/components/Articles.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from "../utils/api";
+import ArticleCard from "./ArticleCard";
+
+function Articles(){
+    const [articles, setArticles] = useState([]);
```

Comment:
Please add a loading state here and let the user know that the articles are loading.